tests/conftest.py

```diff
@@ -103,6 +105,22 @@ def backend(request):
     await send({"type": "http.response.body", "body": b"Hello, world!"})
 
 
+async def server_crash(scope, receive, send):
```

Comment:
Just a suggestion on naming: isn't this endpoint really more of a `server_premature_close`?

Reply:
premature close is more precise, I was naming it after what would typically cause a premature.  I can rename it.